dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -25,8 +25,10 @@
   %script{src: minifiable_asset_path('js/p5play/p5.js')}
   %script{src: minifiable_asset_path('js/p5play/p5.play.js')}
 
-- if use_p5sound
+- if use_dance
```

Comment:
Should we considering switching from gamelab.js + additional dependencies to a separate dance.js entrypoint that has these other libraries included?

Reply:
Updated in b5b34caf2194a3de11575e37f7fe376529e43ab7.  I don't think I can add `p5.sound` to the bundle until we import p5 via npm.  Right now we expect it to be on `window`.